compose/reference/envvars.md

```diff
@@ -4,139 +4,165 @@ keywords: fig, composition, compose, docker, orchestration, cli, reference
+
+* **Default behavior:** If not provided, Compose looks for a file named `docker-compose.yml` in the current directory and, if not found there, Compose then searches each parent directory recursively until a file by that name is found. 
+
+* **Default separator:**  path separators are by default on:
+    * Linux and macOS: `:` (colon), 
+    * Windows: `;` (semicolon). 
+The path separator can also be customized using `COMPOSE_PATH_SEPARATOR`.
 
-This variable supports multiple Compose files separated by a path separator (on
-Linux and macOS the path separator is `:`, on Windows it is `;`). For example:
-`COMPOSE_FILE=docker-compose.yml:docker-compose.prod.yml`. The path separator
-can also be customized using `COMPOSE_PATH_SEPARATOR`.
+Example: `COMPOSE_FILE=docker-compose.yml:docker-compose.prod.yml`. 
 
-See also the `-f` [command-line option](index.md).
+**See also** the [`-f` command-line option](index.md#command-options-overview-and-help).
 
 ## COMPOSE\_PROFILES
 
-Specify one or multiple active profiles to enable. Calling `docker-compose up`
-with `COMPOSE_PROFILES=frontend` will start the services with the profile
-`frontend` and services without specified profiles.
+Specifies one or more profiles to be enabled on `compose-up` execution. 
+Services with matching profiles are started *as well as any services for which no profile has been defined*.
 
-You can specify a list of profiles separated with a comma:
-`COMPOSE_PROFILES=frontend,debug` will enable the profiles `frontend` and
-`debug`.
+For example, calling `docker-compose up`with `COMPOSE_PROFILES=frontend` selects services with the 
+`frontend` profile as well as any services without a profile specified.
 
-See also [_Using profiles with Compose_](../profiles.md) and the `--profile`
-[command-line option](index.md#use---profile-to-specify-one-or-more-active-profiles).
+
+* **Default separator:**  specify a list of profiles using a comma as separator.
+Example: `COMPOSE_PROFILES=frontend,debug`
+This example would enable all services matching both the `frontend` and `debug` profiles *and* services without a profile.
+
+**See also** [Using profiles with Compose](../profiles.md) and the [`--profile` command-line option](index.md#use---profile-to-specify-one-or-more-active-profiles).
 
 ## COMPOSE\_API\_VERSION
 
-The Docker API only supports requests from clients which report a specific
-version. If you receive a `client and server don't have same version` error using
-`docker-compose`, you can workaround this error by setting this environment
-variable. Set the version value to match the server version.
+Specifies the API version of current environment. 
+The Docker API only supports requests from clients reporting a specific version. 
+
+If you receive a **client and server don't have same version** error while using `docker-compose`, 
+you can work around this error by setting this environment variable. 
+Set this version value in order to match the server version.
```

Comment:
Deprecated on V2.

By default the API version is negotiated with the server. To set it, please use `DOCKER_API_VERSION` instead.

Reply:
Added deprecation notice and additional info.